packages/scene-composer/package.json

```diff
@@ -154,10 +154,10 @@
   "jest": {
```

Comment:
should this file be included?

Reply:
Hm, this appears to be autogenerated; maybe I ran the wrong test command at some point. I'll remove.